Handlers/ShowroomHandlers.js

```diff
@@ -68,10 +68,17 @@ function getStats (req, res, next) {
         },
         function (liveResults, inPersonResults, callback) {
             var uniqueUserIDs = new Set();
+            var today = new Date();
+            today.setTime(today.getTime() - 14400000); //Subtract 4 hours (in ms) to account for UTC timezone [needed for production server in ECE]
```

Comment:
Same as above

Reply:
Add the number to config and make it depend on the production variable for compatibility with development

Reply:
added number to config, didn't make it depend on production since it's the same behavior